scipy/interpolate/interpnd.pyx

```diff
@@ -133,8 +133,10 @@ class NDInterpolatorBase(object):
 
         Parameters
         ----------
-        xi : ndarray of float, shape (..., ndim)
+        x1, x2, ... xn: array_like of float
             Points where to interpolate data at.
+            x1, x2, ... xn can be array_like of float with broadcastable shape.
+            or x1 can be array_like of float with shape (..., ndim)
```

Comment:
two minor things:
- `array_like` -> `array-like`
- would be nice to have double backticks around `(..., ndim)`

Reply:
same applies to the other docstring further down

Reply:
Done.